src/main/resources/migrations/00051_add_gateway_account_id_to_payout.sql

```diff
@@ -0,0 +1,6 @@
+--liquibase formatted sql
+
+--changeset uk.gov.pay:add_gateway_account_id_to_payout
+ALTER TABLE payouts ADD COLUMN gateway_account_id VARCHAR(255);
```

Comment:
As this table isn't being used in any environments yet it should be fine to add an index changeset in this migration too. We'll be filtering on `gateway_account_id` so a default index on just this column should be fine.

Reply:
👍